### PR TITLE
delete by id article

### PR DESCRIPTION
**Pull request by IslandRhythms**

### CODE REVIEW

File: src/tutorials.js [hunk 1]
```diff
@@ -5,6 +5,14 @@ const moment = require('moment');
 const axiosPath = require('./axiosPath');
 
 const tutorials = [
+  {
+    title: 'How to Delete By Id in Mongoose',
+    raw: './tutorials/mongoose/deleteById.md',
+    url: '/tutorials/mongoose/deleteById',
```
Comment (vkarpov15) [inline src/tutorials.js:11]: /mongoose/delete-by-id

File: tutorials/mongoose/deleteById.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+```javascript
+const testSchema = new mongoose.Schema({
+    name: String
+});
+
+const Test = mongoose.model('Test', testSchema);
+
+
+async function run() {
+const entry = await Test.create({ name: 'Masteringjs.io' });
```
Comment (vkarpov15) [inline tutorials/mongoose/deleteById.md:14]: Indent

File: tutorials/mongoose/deleteById.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+await Test.find();
+await Test.deleteOne({_id: entry._id});
+await Test.find();
```
Comment (vkarpov15) [inline tutorials/mongoose/deleteById.md:17]: Use `countDocuments()` and add a comment indicating that after `deleteOne()`, the result of `countDocuments()` is 0

File: tutorials/mongoose/deleteById.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+run();
+```
```
Comment (vkarpov15) [inline tutorials/mongoose/deleteById.md:21]: Also add an example of defining a `deleteById()` [instance method](https://mongoosejs.com/docs/guide.html#methods)

**Review by vkarpov15 — CHANGES_REQUESTED** [review]

File: tutorials/mongoose/deleteById.md (hunk 4 above)
Comment (IslandRhythms) [reply tutorials/mongoose/deleteById.md:21]: instance method or static function? From what I read, it seems that `instance methods` are a per document usage while `statics` are for the whole model.

**Review by vkarpov15 — APPROVED** [review]

**Pull request merged**